src/compiler/commandLineParser.ts

```diff
@@ -100,11 +100,12 @@ namespace ts {
                 "umd": ModuleKind.UMD,
                 "es6": ModuleKind.ES2015,
                 "es2015": ModuleKind.ES2015,
+                "es2018": ModuleKind.ES2018

```

Comment:
Though once this is rectify it will then no longer be `ESnext` we will then have to remove from `ESNext` and would that be considered breaking change?  (however, it may be unlikely that this feature will make it in for 2018)